tests/serialization/test_campaign_serialization.py

```diff
@@ -20,11 +20,21 @@ def test_campaign_serialization(campaign):
     assert campaign == campaign2
 
 
-def test_valid_config(config):
+def test_valid_product_config(config):
     Campaign.validate_config(config)
 
 
-def test_invalid_config(config):
+def test_invalid_product_config(config):
     config = config.replace("CategoricalParameter", "CatParam")
     with pytest.raises(ClassValidationError):
         Campaign.validate_config(config)
+
+
+def test_valid_simplex_config(simplex_config):
```

Comment:
As far as I understand this, this uses the specification given in `conftest.py`, so the simplex is created by two parameters using 10 values, right?
In that case, I would prefer to restructure this test: My concern is that we would now not realize if a full space is constructed, since this would probably still be sufficiently effective with the given parameters. I would thus propose to have an example with more parameters such that we can actually see if something goes wrong here.

Reply:
there are separate test (previous PRs) that test whether the simplex creation does what its supposed to

if you have more parameters added, what would that change? It would simply mean the test takes longer, especially in the described error situation, the test would just run long, not fail

Reply:
Yeah, but that is something that we would then see. Even if the test would technically not fail, we would be aware of the error. If we do not include something like that, we would never see that error.

Reply:
but then again: there are tests that test the functionality of the simplex constructor

Reply:
Ok, then I'll trust that these will catch this